src/app/options/options.component.ts

```diff
@@ -31,11 +33,16 @@ export class OptionsComponent implements OnInit {
       });
     }),
   );
+  additionalOptions: string = "";
 
   constructor() {}
 
   ngOnInit(): void {}
 
+  obabelOptionsString(): string {
```

Comment:
What purpose does this string serve? If this is being added with a purpose, please add a test for it.